plugins/forc-index/src/ops/forc_index_build.rs

```diff
@@ -64,6 +64,8 @@ pub fn init(command: BuildCommand) -> anyhow::Result<()> {
         );
     }
 
+    let path = path.unwrap_or(".".into());
```

Comment:
Can we use `std::env::current_dir()` instead of `".".into()` as it's platform-agnostic?